monero-outreach-docs/translations/es/archive/rpc_pay_es.md

```diff
@@ -1,113 +1,109 @@
 
-A modern family of RPC interface software uses the language of JavaScript Object Notation (JSON) to form messages. JSON is a standard textual format for exchanging data. It relies on name-value pairs and lists of data to define information in a way that is robust, extendable, and human readable. JSON encoding is the primary method used by Monero’s RPC framework.
+Un grupo moderno de la familia de software de interfaz RPC utiliza el lenguaje de JavaScript de notación de objetos (JSON) para formar mensajes. JSON es un formato de texto estándar para intercambiar información. Se basa en el par "nombre-valor" y listas de datos que definen la información de una manera amplia, extendible y legible par las personas. La codificación JSON es el principal método usado por el esquema RPC de Monero.
 
-For Monero, RPC is already being used in a variety of ways. For example, the standard Monero daemon, containing most of Monero's functionality, is _monerod_. It implements the server side of RPC for a variety of blockchain-related function calls. And the program _monero-wallet-rpc_, delivered with the standard Monero installation, implements the server side of wallet functionality while itself being a client to a _monerod_ server. Monero RPC calls work much like a browser communicating with a web server. Data is sent using the POST method, and returned information is textual, usually JSON. If you are interested in detailed examples of RPC messages, please see [Example #1](https://www.monerooutreach.org/stories/RPC-Pay.html#box1) on using Monero RPC calls.
+Para Monero, el RPC ya está siendo utilizado de muchas maneras. Por ejemplo, el _monerod_, es el daemon estándard que contiene la mayoría de funcionalidades de Monero. Implementa el lado del servidor del RPC para una variedad de funciones relacionadas a las llamadas a la cadena de bloques. Y el programa _monero-wallet-rpc_, que se entrega con la instalación estándard de Monero, implementa el lado del servidor de las funciones del monedero, mientras que al mismo tiempo es un cliente a un servidor _monerod_. Las llamadas RPC de Monero trabajan muy parecido a la comunicación entre un navegador (browser) y un servidor web. Las información es envíada usando el método POST y devuelta como texto, usualmente JSON. Si estás interesado en ejemplos detallados en el uso de las llamadas RPC de Monero, mira el [Ejemplo #1](https://www.monerooutreach.org/stories/RPC-Pay.html#box1). 
```

Comment:
A little rephrase as suggestion:

_Por ejemplo, **_monerod_ es el daemon** estándard..._

_Implementa el lado del servidor del RPC para una **variedad de funciones de llamadas relacionadas a la cadena de bloques._

And a correction:

_**La** información es **enviada** usando el método POST y devuelta como texto, usualmente JSON. Si estás interesado en ejemplos detallados en el uso de las llamadas RPC de Monero, mira el [Ejemplo #1](https://www.monerooutreach.org/stories/RPC-Pay.html#box1) **sobre usar llamadas RPC de Monero**._

For _**envíada**_ I know some sites say it should have accent, but as per the most recent RAE information, it doesn't. You can see that [here](https://dle.rae.es/enviado).

Reply:
Thank you @BlackLotus64, you're right. Fixed and changed.